webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtVmware.js

```diff
@@ -48,6 +48,41 @@ const VirtVmware = ({ vm }) => (
         {vm.corespersocket}
       </DescriptionListDescription>
     </DescriptionListGroup>
+    {Object.values(vm.disks).map((disk, index) => (
+      <DescriptionListGroup key={`disk-${index}`}>
+        <DescriptionListTerm>{__('Disk capacity')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {NumberToHumanSize(disk.capacity * 1024 * 2, {
```

Comment:
Why `* 2`? Did you intend `** 2`? If so, I'd still write it in long form:
```suggestion
          {NumberToHumanSize(disk.capacity * 1024 * 1024, {
```

Probably also below just because `* 1024 * 1024` is easier to read than `* 1024 ** 2`. Or define `const mb = 1024 * 1024` and then use `disk.capacity * mb` for maximum readability.

Reply:
Sorry, "**" was intended. I re-used it from existing code above.